test/vms/ramen/etc/init.d/increase-root-filesystem-size

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+LOGFILE="/var/log/resize-filesystem.log"
+
+case "$1" in
+    start)
+        echo "Starting resize..." | tee -a "$LOGFILE"
+        df -h / >> "$LOGFILE"
+        DEVICE=$(df -h / | grep -vE '^Filesystem|tmpfs|cdrom' | awk '{print $1}')
```

Comment:
To find the device it is better to use lsblk. Parsing df text output is hard to understand and maintain and is less reliable.

Reply:
sure wiill make the change

Reply:
i checked lsblk does not provide the required disk i feel df -h is the right option for now

Reply:
found a different way i have updated the script